cuegui/cuegui/DarkPalette.py

```diff
@@ -126,3 +126,10 @@ def ColorF(r, g, b):
 COLOR_SHOW_BACKGROUND = GreyF(0.13)
 COLOR_SHOW_FOREGROUND = GreyF(0.79)
 COLOR_JOB_FOREGROUND = GreyF(0.79)
+
+#Log file Colors
+LOG_TIME = QtGui.QColor(170, 149, 171)
+LOG_ERROR = QtGui.QColor(224, 52, 52)
+LOG_WARNING = QtGui.QColor(255, 201, 25)
+LOG_INFO = QtGui.QColor(111, 140, 255)
+LOG_COMPLETE = QtGui.QColor(132, 201, 12)
```

Comment:
nit: There should be a newline at the end of every code file. There should be an IDE setting you can use to do this automatically.